src/boosting/gbdt.h

```diff
@@ -557,18 +557,18 @@ class GBDT : public GBDTBase {
   score_t* gradients_pointer_;
   /*! \brief Pointer to hessian vector, can be on CPU or GPU */
   score_t* hessians_pointer_;
-  /*! \brief Whether boosting is done on GPU, used for cuda_exp */
+  /*! \brief Whether boosting is done on GPU, used for cuda */
```

Comment:
Change `cuda` to `device=cuda` for clarity.

Reply:
sure! Although since `device` is just an alias for main parameter `device_type` (https://lightgbm.readthedocs.io/en/latest/Parameters.html#device_type), I think we should use `device_type`.

```suggestion
  /*! \brief Whether boosting is done on GPU, used for device_type=cuda */
```